oak/server/rust/oak_runtime/src/node/grpc/server/mod.rs

```diff
@@ -290,6 +290,11 @@ impl Service<http::Request<hyper::Body>> for HttpRequestHandler {
 
         let method_name = request.uri().path().to_string();
         let metrics_data = self.runtime.metrics_data();
+        metrics_data
+            .grpc_server_metrics
+            .grpc_server_msg_received_total
+            .with_label_values(&[&method_name])
+            .observe(1.0);
```

Comment:
#97 should encompass bidir, but it's unlikely to happen any time soon &ndash; partly because we're not aware of an immediate need for it, partly because it's hard :-).

Reply:
Then I'll remove it for now :)